docs/user/dashboard/lens.asciidoc

```diff
@@ -48,389 +13,144 @@ To view the data summary for a field, click *i* next to the field.
-image::images/lens_end_to_end_6_1.png[Custom ranges configuration]
-
-.. From the *Value format* dropdown, select *Bytes (1024)*, then click *Close*.
-
-. From the *Chart Type* dropdown, select *Pie*.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_6_2.png[Files size distribution]
-
-. *Save* the panel, enter `File size distribution` in the *Title* field, then add the panel to the dashboard.
-
-[discrete]
-[[treemap]]
-=== View the top sources of website traffic
-
-To determine how users find out about your website and where your users are located, create a treemap that displays the percentage of users that 
-enter your website from specific social media websites, and the top countries where users are located.  
-
-. Open *Lens*.
-
-. From the *Chart Type* dropdown, select *Treemap*.
-
-. From the *Available fields* list, drag and drop *Records* to the *Size by* field in the editor. 
-
-. From the editor, click the *Drop a field or click to add* field for *Group by*, then create a filter for each website traffic source.
-
-.. From *Select a function*, click *Filters*.
+image::images/lens_bucketed_aggregation_advanced_dropdown.png[Lens Advanced options for bucketed aggregations]
 
-.. Click *All records*, enter the following, then press Return:
-
-* *KQL* &mdash; `referer : *facebook.com*`
-
-* *Label* &mdash; `Facebook`
-
-.. Click *Add a filter*, enter the following, then press Return:
-
-* *KQL* &mdash; `referer : *twitter.com*`
-
-* *Label* &mdash; `Twitter`
-
-.. Click *Add a filter*, enter the following, then press Return:
-
-* *KQL* &mdash; `NOT referer : *twitter* OR NOT referer: *facebook.com*`
-
-* *Label* &mdash; `Other`
-
-.. Click *Close*.
+[float]
+[[is-it-possible-to-select-color-for-specific-bar-or-point]]
+===== How do I change the color for a single data point?
 
-[discrete]
-[[add-the-countries]]
-==== Add the geographic data
+*Lens* provides you with color pallettes that you can apply to the entire visualization, but you are unable to change the color for a single data point, such as a bar or line.
 
-To determine the top countries where users are located, add the geographic data.
+[float]
+[[can-i-sort-by-multiple-columns]]
+===== How do I sort by multiple columns?
 
-Compare the top sources of website traffic data to the top three countries. 
+Multiple column sorting is unsupported in *Lens*, but is supported in *Discover*. For information on how to sort multiple columns in *Discover*, 
+refer to <<explore-fields-in-your-data,Explore the fields in your data>>.
 
-. From the *Available fields* list, drag and drop *geo.src* to the visualization builder. 
+[float]
+[[is-it-possible-to-inspect-elasticsearch-queries]]
+===== How do I inspect {es} queries?
 
-. To change the *Group by* order, click and drag *Top values of geo.src* so that it appears first in the editor.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_7_2.png[Treemap vis]
+Inspecting {es} queries in visualizations is unsupported in *Lens*. 
```

Comment:
Need to update this with the `Inspector` flow: `Inpector` => `View: Request` => `Request ....`?

Reply:
We already cover how to inspect visualizations on the Dashboard page, so this isn't needed.